### DISCUSSION TITLE
When I favourite a song (not an album), what possibilities does that enable?

### DISCUSSION THREAD

**Discussion by IO101010** [navidrome/navidrome#3278] [Q&A]
I’ve hearted some songs I like within various albums, but when I click on the favourites view (in the left menu bar) it is empty. 

I figured out that only favourited albums appear there. So where do I view my favourited songs? I’d like to listen to them as a collection of some kind.

**Comment by deluan** [comment]
The "Favourites" in Navidrome UI is just for albums, not songs. You can create a [Smart Playlist](https://github.com/navidrome/navidrome/issues/1417) with just your favourites:
```
{
  "all": [
    {"is": {"loved": true}}
  ],
  "sort": "dateLoved",
  "order": "desc"
}
```

Also, most Subsonic clients show your Favourite songs in some way. Examples (DSub and Symfonium):
<p align="left">
    <img height="700" src="https://github.com/user-attachments/assets/2a361672-c741-4a87-836d-b0dee33b20bf">
    <img height="700" src="https://github.com/user-attachments/assets/0c02f1fd-ccaf-40c1-8ff5-74057e86df95">
</p>

**Reply by IO101010** [reply]
Thanks. I also see that I can go to the "Songs" view and filter by "hearted".
